### PR TITLE
docs: seo optimisations

### PR DESCRIPTION
**Pull request by matthewelwell**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [ ] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

Adds 2 changes to the docs site as requested by SEO agency. 

 1. Add 'feature flag tool' to the intro statement
 2. Adds all our how-to guides as links in the footer

I don't love the aesthetic of the how-to guides in the footer, but if it's helpful for SEO then we should probably just swallow it for now. I can't see any obvious options to split the columns, so we do end up with one large column of all of the guides. 

image

## How did you test this code?

Ran the docs site locally and verified the changes.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -119,6 +119,63 @@ const config = {
                             },
                         ],
                     },
+                    {
+                        title: 'How-to guides',
+                        items: [
+                            {
+                                label: 'Angular Feature Flags',
```
Comment (rolodato) [inline docs/docusaurus.config.js:126]: Is it necessary to have "Feature Flags" on each label? It looks visually noisy IMO but if there's truly a SEO benefit I'm guessing it's okay.

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -119,6 +119,63 @@ const config = {
...
+                            {
+                                label: 'Java Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/java-feature-toggle',
+                            },
+                            {
+                                label: 'Ruby Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/ruby-feature-flags',
+                            },
+                            {
+                                label: 'Javascript Feature Flags',
```
Comment (rolodato) [inline docs/docusaurus.config.js]:
```suggestion
                                label: 'JavaScript Feature Flags',
```

JavaScript is the correct capitalisation :)

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -119,6 +119,63 @@ const config = {
                             },
                         ],
                     },
+                    {
+                        title: 'How-to guides',
```
Comment (rolodato) [inline docs/docusaurus.config.js:123]: Is the order for these links intentional? I'm curious why Angular is first but more popular platforms like iOS, React Native etc are lower.

I don't have a strong preference either way - maybe alphabetical order makes it easier to skim through the list and makes it easier to decide what goes first.

**Review by rolodato — APPROVED** [review]
Left some thoughts, nothing worth blocking the PR 👍

File: docs/docusaurus.config.js (hunk 3 above)
Comment (dabeeeenster) [reply docs/docusaurus.config.js:123]: Yes I think alphabetical here?

File: docs/docusaurus.config.js (hunk 1 above)
Comment (matthewelwell) [reply docs/docusaurus.config.js:126]: I think there probably is an SEO benefit here, but I will check on this as it's a good point.

**Pull request merged**